paddle/fluid/operators/mkldnn/mul_mkldnn_op.cc

```diff
@@ -207,6 +207,13 @@ class QuantMulPrimitiveFactory : public MulPrimitiveFactory<XT, YT, OT> {
     int y_num_col_dims = ctx.Attr<int>("y_num_col_dims");
     auto scale_y = ctx.Attr<std::vector<float>>("scale_y");
 
+    // TODO(minghui) : to make int8 mul op more general.
```

Comment:
This TODO is confused. What's the meaning of `more general`?

Reply:
`TODO(minghui)`->`TODO(intel-minghui)`, for `grep` and statistics easily

Reply:
done